File: Type-3/JoseVitorSAmorim_2.js
const prompt = require('prompt-sync')();
while (true) {
    let nome = prompt('Nome: ');
    let velocidade_permitida = parseFloat(prompt('Velocidade Permitida: '));
    let velocidade_real = parseFloat(prompt('Velocidade Real: '));
    if (velocidade_permitida>0 && velocidade_real>0) {
        let percentual = (velocidade_real - velocidade_permitida) / velocidade_permitida * 100;
        if (velocidade_real <= velocidade_permitida) {
            console.log(nome + ' não receberá multa.');
        }
        else if (percentual <= 20) {
            console.log(nome + ' receberá multa leve.');
        }
        else if (percentual <= 50) {
            console.log(nome + ' receberá multa grave.');
        }
        else if (percentual > 50) {
            console.log(nome + ' receberá multa gravíssima.');
        }
    }
    else {
        console.log('Calma ai patrão, tem algo errado.');
        console.log('Tente novamente.');
        console.log('-'.repeat(30));
    }
    let opcao = prompt('Deseja continuar[S/N]? : '.toLowerCase());
    if (opcao !== 's') {
        break;
    }
}
console.log('-'.repeat(30));
console.log('Programa encerrado.');

File: Desen_Sis&Back-End/Type-3/JoseVitorSAmorim_2.java
import java.util.Scanner;
public class JoseVitorSAmorim_2 {
    public static void main(String[] args) throws InterruptedException {
        Scanner scanner = new Scanner(System.in);

        while (true) {
            System.out.print("Nome: ");
            String nome = scanner.nextLine();
            System.out.print("Velocidade permitida: ");
            float velocidadePermitida = scanner.nextFloat();
            System.out.print("Velocidade real: ");
            float velocidadeReal = scanner.nextFloat();
            if (velocidadePermitida > 0 && velocidadeReal > 0) {
                float percentual = ((velocidadeReal - velocidadePermitida) / velocidadePermitida) * 100;

                if (velocidadeReal <= velocidadePermitida) {
                    System.out.println(nome + " não receberá multa");
                }
                else if (percentual <= 20) {
                    System.out.println(nome + " receberá multa leve");
                } 
                else if (percentual <= 50) {
                    System.out.println(nome + " receberá multa grave");
                } 
                else {
                    System.out.println(nome + " receberá multa gravíssima");
                }
            } 
            else {
                System.out.println("\033[31mCalma ai patrão, tem algo errado\033[m");
                System.out.println("\033[33mTente novamente\033[m");
                System.out.println("-".repeat(30));
            }
            scanner.nextLine();
            System.out.print("Deseja continuar[S/N]: ");
            String opcao = scanner.nextLine().toUpperCase();
            if (opcao.equals("N")) {
                break;
            }
        }
        System.out.println("\033[33mFinalizando o programa...\033[m");
        Thread.sleep(2000);
        System.out.println("\033[32mObrigado por usar nosso sistema\033[m");
        scanner.close();
    }
}
